ads/google/a4a/traffic-experiments.js

```diff
@@ -49,30 +53,37 @@ export let ExperimentInfo;
  * @param {!Window} win  Host window for the ad.
  * @param {!Element} element Ad tag Element.
  * @param {string} experimentId ID for the experiment to use.
- * @param {!Branches} experiment and control branch IDs to use.
+ * @param {!Branches} externalBranches experiment and control branch IDs to use
+ *   when experiment is triggered externally (e.g., via Google Search
+ *   results page).
+ * @param {!Branches} internalBranches experiment and control branch IDs to
+ *   use when experiment is triggered internally (i.e., via client-side
+ *   selection).
  * @return {boolean}  Whether Google Ads should attempt to render via the A4A
  *   pathway.
  */
-export function googleAdsIsA4AEnabled(win, element, experimentId, branches) {
+export function googleAdsIsA4AEnabled(win, element, experimentId,
+    externalBranches, internalBranches) {
   if (isGoogleAdsA4AValidEnvironment(win)) {
     // Page is served from a supported domain.
```

Comment:
I think we can remote this line of comment. It's quite readable already in the if check.